fendermint/vm/interpreter/src/fvm/state/exec.rs

```diff
@@ -369,22 +359,6 @@ where
         f(&mut self.params);
         self.params_dirty = true;
     }
-
-    pub fn call_state(&self) -> anyhow::Result<FvmCallState<DB>> {
-        let externs = self.executor.externs().read_only_clone();
```

Comment:
@cryptoAtwill can we also clean up and remove the read_only_clone() method?

Reply:
removed!